terraform/providers.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0.0"
+      version = "~> 5.19.0"
```

Comment:
This looks like _way too big_ of an upgrade to blithely apply. Micro versions I'm fine with; minor versions make me nervous, but a _major_ version? Forget it!